avahi-sharp/EntryGroup.cs

```diff
@@ -41,7 +41,7 @@ public enum PublishFlags {
     }
 
     public enum EntryGroupState {
-        Uncommited,
+        Uncommitted,
```

Comment:
(see also other comment on API change) This would also change (potentially) the API of current C# programs using Avahi, so would therefore need to have its own separate PR. (Maybe needs to be included in an API revision or so.)